index.html

```diff
@@ -4,162 +4,89 @@
+              <a class="detailBorder pb-2" href="">READ MORE</a>
             </div>
           </div>
-          <div class="content content--vertical">
-            <div class="imageBox imageBox--half">
-              <img
-                class="imageBox__image"
-                src="./img/fruitInPlate.png"
-                alt="tomato, beet and apple in a white plate"
-              />
-            </div>
-            <div class="imageBox imageBox--half">
-              <img
-                class="imageBox__image"
-                src="./img/asparagus.png"
-                alt="a few fresh asparagus"
-              />
+          <div class="col-md-4">
+            <img
+              class="mw-100"
+              src="./img/christmasMeal.png"
+              alt="christmasMeal"
+            />
+            <h6 class="pt-4 textBold lineHeight30">
+              A Traditional Christmas Eve, Mark Best Style
+            </h6>
+            <p class="textSilver lineHeight28 pt-1">
+              One of Australia's best chefs and AEG ambassador, Mark Best,
+              shares his favourite Christmas Eve menu which is sure to impress
+              your guests.
+            </p>
+            <div class="pt-3 pb-5">
+              <a class="detailBorder pb-2" href="">READ MORE</a>
             </div>
           </div>
-          <article class="content content--vertical">
-            <div class="textBox">
-              <a
-                class="textBox--no-decoration"
-                href="http://0rz.tw/AaW9E"
-                target="_blank"
-              >
-                <h1
-                  class="textBox__head textBox__head--grey textBox__head--upper"
-                >
-                  Answer your body's needs
-                </h1>
-              </a>
-              <hr />
-              <p class="textBox__desc">
-                The way ingredients are sourced affected the way we nourish our
-                bodies, Author Mark Schatzer believes our body naturally
-                develops an appetite for the foods and nutrients it needs to be
-                healthy, but that artificial flavouring are getting in the way.
-                This can be reversed by focusing on high-quality ingredients and
-                being mindful as your appetite guides you to consume according
-                to your body's needs.
-              </p>
-              <h4
-                class="textBox__head textBox__head--red textBox__head--appendix textBox__head--upper"
-              >
-                be mindful
-              </h4>
-              <p class="textBox__desc textBox__desc--white">
-                Sourcing local or organic food is a good way to start being more
-                mindful about what you're cooking and eating.
-              </p>
+          <div class="col-md-4">
+            <img class="mw-100" src="./img/salad.png" alt="salad" />
+            <h6 class="pt-4 textBold lineHeight30">Taking Taste Further</h6>
```

Comment:
Just another thought. Think throughoutly before you decide to do so. 

Did you notice it's not DRY? `pt-4 textBold lineHeight30` It's possible to make these bootstrap utility class names to a single semantic class name. 

```
@import 'bootstrap.css';

.section-title {
  @extend .pt-4;
  @extend .font-weight-bold;
  line-height: 30px;
}

<h6 class="section-title">...</h6>
```

Reply:
Sometimes it is more clear for developer with all the class shows inside html.
I would give class name for repeat elements in html but won't give class name for all elements.
Is that make sense to you?

Reply:
Sure, as long as you can explain your philosophy. 😉